Automattic/pocket-casts-ios
# Tracks: Add chromecast events

| 📘 Project: #154 |
|:---:|

- `chromecast_view_shown`: When the chromecast view is shown
- `chromecast_started_casting`: When the user taps a device and starts casting to a device
- `chromecast_stopped_casting`: When the user taps the stops casting
- `chromecast_view_dismissed`: When the user dismisses the chromecast view

## To test

1. Launch the app
2. Tap on the podcast tab
3. Select a podcast
4. Tap the Chromecast icon in the top right
5. ✅ `🔵 Tracked: chromecast_view_shown ["is_connected": false]`
6. Tap on a device to start casting
7. ✅ `🔵 Tracked: chromecast_started_casting`
8. Verify you also see: `🔵 Tracked: playback_play ["source": "chromecast"]`
9. Tap the chromecast icon again
10. ✅ `🔵 Tracked: chromecast_view_shown ["is_connected": true]`
11. Tap the pause button
12.  ✅  `🔵 Tracked: playback_pause ["source": "chromecast"]`
13. Tap the play button
14. ✅   `🔵 Tracked: playback_play ["source": "chromecast"]`
15. Tap the stop casting button
16. ✅  `🔵 Tracked: chromecast_stopped_casting`

## Checklist
